src/main/java/tempest/GUI/ManageModulesPage.java

```diff
@@ -0,0 +1,89 @@
+package tempest.GUI;
+
+import tempest.GUI.components.BackButton;
+import tempest.GUI.components.ModuleDropDown;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+
+public class ManageModulesPage extends Page {
+    private final GUIManager manager;
+    private final ModuleDropDown moduleDropDown = new ModuleDropDown();
+
+    private JPanel manageModulesPanel;
+    private JPanel pageSwapPanel;
+    private JPanel backPanel;
+
+    private JButton addModuleButton;
+    private JButton deleteModuleButton;
+    private BackButton backButton;
+    private JComboBox<Object> dropDown;
+
+    public ManageModulesPage(GUIManager guiManager){this.manager = guiManager;}
+
+    @Override
+    public String getName() {
+        return "manageModulesPage";
+    }
+
+    public JPanel getPanel() {
+        manageModulesPanel = new JPanel();
+        pageSwapPanel = new JPanel();
+        backPanel = new JPanel();
+
+        dropDown = moduleDropDown.getModuleDropDown();
+
+        addModuleButton = new JButton("Add a module");
+        deleteModuleButton = new JButton("Delete a module");
+        backButton = new BackButton(manager);
+
+
+        addModuleButton.addActionListener(this);
+        deleteModuleButton.addActionListener(this);
+        backButton.addActionListener(this);
+
+        pageSwapPanel.add(addModuleButton);
+        pageSwapPanel.add(deleteModuleButton);
```

Comment:
This is stupidly picky, but... atm we have the 'positive' button on the right (e.g. "Enter"), whilst the 'negative' button is on the left (e.g. "Back")

So if you don't mind could you just swap the order these two added to the panel